### PR TITLE
project-weather-app jonna

### PR DESCRIPTION
**Pull request by jonnaru**
**Link to my deployed project:**
https://hello-weather.netlify.app/

**My reflections on how this week's project turned out:**
This week has been really good. It was fun coding an app and learn about fetch and API.

### CODE REVIEW

**Review by ssundblom — COMMENTED** [review]
WOW! I'm so impressed by your project! Black level all the way! 🖤
Your page is super stylish and I love the gradient background, the pulsating icons! 
It feels super user friendly and is comfortable for the eyes ☺️ It looks really amazing! 
The way you've structured your code makes it very easy to read and your comments makes it very clear with what you wanted to accomplish.  I especially like your clean and simple functions! 
As I said, super impressed! Keep up the good work! Will for sure save this page for the days when the inspiration is running dry! :)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+ul {
+  box-sizing: border-box;
+  list-style: none;
+  padding: 20px;
+  text-align: center;
+  margin: 0 auto;
+  width: 80%;
+  /* border: 4px solid #fff; */
+  /* border-style: hidden solid solid solid; */
+  border-radius: 10px;
```
[the remark this answers is not in the document]
Comment (jonnaru) [reply style.css]: Thank you! It is now deleted 🌼

File: style.css [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+    transform: scale(1);
+  }
+}
+
+#mainWeather {
+  animation-name: pulse;
+  animation-duration: 5s;
+  animation-iteration-count: infinite;
+}
+
```
[the remark this answers is not in the document]
Comment (jonnaru) [reply style.css:159]: Haha! Great :)))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```
[the remark this answers is not in the document]
Comment (jonnaru) [reply index.html]: It's good to be picky!

File: script.js [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+  "very heavy rain",
+  "extreme rain",
+];
+
+let heavyRain = [
+  "light intensity shower rain",
+  "shower rain",
+  "heavy intensity shower rain",
+  "ragged shower rain",
+];
```
[the remark this answers is not in the document]
Comment (jonnaru) [reply script.js]: Noooo... 😳 They are all now const. Thank you!

File: script.js [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+  fetch(apiURLcurrent)
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      // *** city
+      city.innerHTML = data.name;
+
+      console.log(data);
+      // *** temperature
```
[the remark this answers is not in the document]
Comment (jonnaru) [reply script.js]: Done :)

**Comment by ssundblom** [comment]
DAMN YOU ARE FAST! 🌟
Good job! And seriously, your page is amazing! Be proud  👸🏼

Den mån 28 sep. 2020 kl 21:28 skrev Jonna Ru <notifications@github.com>:

File: index.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+    <header>
+
+      <div class="header-info">
+        <p id="weatherToday" class="weather-today"></p>
+        <p>sunrise <span id="sunRise"></span></p>
+        <p>sunset <span id="sunSet"></span></p>
+      </div>
+
+      <h2 id="loading"></h2>
```
Comment (EmOlsz) [inline index.html:30]: Great job with this loader :)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    </header>
+
+    <main id="main">
+
+      <img id="mainWeather" class="main-weather" height="240" src="" alt="weather icon">
+      <h1><span id="city"></span> <span id="temp"></span>°C</h1>
```
Comment (EmOlsz) [inline index.html:36]: All of those elements should be in separate lines

File: index.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+        <li id="newYork" onclick="changeCity('New York')">New York</li>
+        <li id="london" onclick="changeCity('London')">London</li>
+        <li id="moscow" onclick="changeCity('Moscow')">Moscow</li>
+      </ul>
+
+    </main>
+
+  </div>
+
+  <script src="script.js"></script>
```
Comment (EmOlsz) [inline index.html:59]: `./`

File: script.js [hunk 9]
```diff
@@ -0,0 +1,351 @@
...
+};
+
+// from unix time to milliseconds
+const time = (number) => {
+  return new Date(number * 1000);
+};
+
+////////////////////////////////////////////////////////////////////////////
+
+// *** CURRENT WEATHER
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply script.js:64]: Truly!

File: script.js [hunk 10]
```diff
@@ -0,0 +1,348 @@
...
+
+      // *** sunset
+      sunset = time(data.sys.sunset + timeZone + offset);
+      // "short" makes time show 4 numbers e.g. "18:30"
+      sunSet.innerHTML = `${sunset.toLocaleTimeString([], {
+        timeStyle: "short",
+      })}`;
+
+      // day-time
+      const isDay = () => {
```
Comment (EmOlsz) [inline script.js:106]: Very clean and well done function!

**Review by EmOlsz — APPROVED** [review]
Nice, well done project with clean functions, great job! Also, a lot of extra requirements are done, I'm super happy to see that Jonna :)
Take a look on my comments though, they are minor, yet important for future projects :)

**Pull request closed without merging**